[TW-1201] CSV file formatting (#4885)
postmanlabs/postman-docs: src/pages/docs/collections/running-collections/working-with-data-files.md

You may encounter errors when Postman attempts to read your data file during a collection run. If this happens, you can take the following steps:

* Ensure your data file is formatted as either CSV or JSON.

> * If you want to keep very large numbers or numbers with leading zeros, specify them as a string in the CSV file by wrapping the number in double quotes as follows:

Reviewer: ```suggestion
* If you want to keep large numbers or numbers with leading zeros, specify them as a string in the CSV file by wrapping the number in double quotes as follows:
```

Reviewer: Vale suggests not using "very" 😀

  ```csv
  index,specialNumber
  1,"000001"
  2,"9223372036854775807"
  ```

> * Confirm your data file is encoded properly, by opening your data file in a text editor and saving the file in a different format.

Reviewer: ```suggestion
* To ensure the correct encoding of your data file, open the file in a text editor and save it in a different format.
```

Reviewer: Vale also doesn't like "properly". Here's one way you could reword things if you wanted to avoid it
